src/index.ts

```diff
@@ -0,0 +1,17 @@
+export { Trace, ITraceContext, TraceContext, useTrace } from './analytics/Trace.js'
+export { TraceEvent } from './analytics/TraceEvent.js'
+
+export {
+  BROWSER,
+  CUSTOM_USER_PROPERTIES,
+  ElementName,
+  Event,
+  EventName,
+  ModalName,
+  PageName,
+  SWAP_PRICE_UPDATE_USER_RESPONSE,
+  SectionName,
+  WALLET_CONNECTION_RESULT,
+} from './analytics/constants.js'
```

Comment:
Right now we have a lot of different constants for pieces of data like `EventName`, `ModalName`, etc. It seems like some of these are ripe for grouping/nesting, which both clarifies some structure of them as well as minimizing top level exports. Thoughts on that?

In general these are a lot of different "types" of constants, so adding that exported nesting may be easier / lighter weight than say doing separate files and having different export paths exposed to developers

Reply:
resolved offline!